ImageConverter/ImageConverter/Input.cs

```diff
@@ -0,0 +1,105 @@
+﻿using ImageConverter.Bmp;
+using ImageConverter.Ppm;
+
+namespace ImageConverter
+{
+    internal class Input
+    {
+        public Input()
+        {
+
+        }
+
+        public void InputCheck(string[] args)
+        {
+            string source = "";
+            string destination = "";
+            string goal_format = "";
+            bool isDestinationParametеrExist = false;
+            bool isSourceParameterExist = false;
+            bool isGoalFormatParameterExist = false;
+
+            if (args.Length > 6)
+            {
+                throw new Exception("Too many parameters");
+            }
+
+            for (int i = 0; i < args.Length; i++)
+            {
+                if (args[i] == "--destination")
+                {
+                    destination = args[i + 1];
+                    isDestinationParametеrExist = true;
+                }
+                if (args[i] == "--source")
+                {
+                    source = args[i + 1];
+                    isSourceParameterExist = true;
+                }
+                if (args[i] == "--goal-format")
+                {
+                    goal_format = args[i + 1];
+                    isGoalFormatParameterExist = true;
+                }
+            }
+
+            if (!isDestinationParametеrExist)
+            {
+                throw new Exception("you forget --destination");
```

Comment:
It'd be better to throw an exception based on not only one problem but on all of them with a message containing all encountered input mistakes